lib/pack/constants.go

```diff
@@ -44,8 +44,10 @@ const (
 	PurposePlanetConfig = "planet-config"
 	// PurposeRuntime marks a package as a runtime container package
 	PurposeRuntime = "runtime"
-	// PurposeTeleportConfig marks packages with teleport config
+	// PurposeTeleportConfig marks package with teleport master config
 	PurposeTeleportConfig = "teleport-config"
+	// PurposeTeleportNodeConfig marks package with teleport node config
```

Comment:
I only added the node teleport configuration marker - master configuration is still generated using the old label.